docs/reference/concepts/hyperscaling.md

```diff
@@ -1,152 +1,205 @@
+### Economic Guarantees
 
-Synchronicity is important as it enables atomic transactions between partitions, unlocking several unique use-cases:
+The time it takes to hyperbridge is the settlement time of the proof, which can only happen after the proof is generated. This will be on the order of minutes, but for some chains faster interoperability will be needed. This is impossible to achieve trustlessly, but it is easy to achieve if have economic guarantees. The transaction root can be calculated outside the proof and then imported. This is not as secure as importing the transaction root from a proof, even a single invalid transaction means that all of the participating rollups will have to revert, as valid proofs cannot be generated. So we are building this solution as an optional add-on, and not part of the core protocol. 
 
-- Transparently reading the state of another partition.
-- Using flash loans between the partitions.
+For these rollups proof generation and settlement still happens as usual. This means the transaction root will have to be calculated inside the L2 or specialised proof. This means this add-on can only work for L3s or for the Layered Aggregator.  
 
-One prominent example of this is a combination of [zkRollup + zkPorter](https://blog.matter-labs.io/zkporter-a-breakthrough-in-l2-scaling-ed5e48842fbf) (which will be part of the zkSync Basechain):
+<div  align="center">
 
-![Logical state partitions!](../../assets/images/hyperscaling3.png "Logical state partitions")
+![](../../assets/images/hyperscalingFastEconomic.png)
 
-<!-- ### MEV protection
+</div>
 
-We expect Hyperchain developers to experiment with different MEV-minimization approaches (see the [question about MEV in zkSync](https://docs.google.com/document/d/1Oq-grdnlY8RL8aoA1EAiRPlJSbH698BoTVK-5bX8nxc/edit#heading=h.5ygkd14evhl0)). -->
+### Atomicity
 
-### Privacy
+Another property that we would like to enable long-term is atomicity for the transactions. This can be achieved using the DA layer. Instead of a relayer sending the transaction to the destination rollup, it can be posted to the DA layer, and the receiving rollup can read it from there. To guarantee atomicity, the received transactions need to be compared to the sent transactions. If rollups generated blocks at the same frequency then this could be done from block to block inside the proof aggregator. Unfortunately this is not the case, rollups will choose to generate blocks and setle proofs at their own rate. This makes the mechanism more complicated, a detailed description is available [here](https://hackmd.io/@kalmanlajko/BkmTTADai), and is still under review. This will only be possible for the Layered Aggregator. 
+
+<div  align="center">
+
+![ ](../../assets/images/hyperscalingAtomicityDA.png)
+This image is an intuition pump, the real mechanism is more complicated and works differently. 
```

Comment:
shall we remove this phrase or amend it to something like "is a high level overview"